client/src/components/network-diagram/index.tsx

```diff
@@ -66,64 +61,86 @@ const NetworkDiagram = ({
       <div className="mb-6 font-serif text-2xl text-slate-700">Network</div>
       <div className="flex flex-col-reverse justify-between gap-4 xl:flex-row">
         <div className="-mt-4 flex-grow overflow-hidden">
-          <Item name={name} id={id} type={type} hasChildren={networks?.length > 0} />
-          {networks.map((network, childIndex) => {
-            if (
-              !network ||
-              typeof network === 'undefined' ||
-              typeof network.name === 'undefined' ||
-              typeof network.id === 'undefined'
-            ) {
-              return null;
-            }
-
-            return (
-              <Collapsible key={network.id} className="ml-10">
-                <Item
-                  key={network.id}
-                  name={network.name}
-                  type={network.type as 'organization' | 'project'}
-                  category={network.category}
-                  id={network.id}
-                  onToggle={(open) => handleOnToggle(open, network.id, setOpenCollapsibles)}
-                  opened={openCollapsibles.includes(network.id)}
-                  heightIndex={getIndex(childIndex)}
-                  hasChildren={network?.children?.length > 0}
-                  isFirstOfType={childIndex === 0}
-                />
-                <CollapsibleContent className="ml-10">
-                  {network?.children.map(
-                    (child, grandChildIndex) =>
-                      child &&
-                      typeof child.id !== 'undefined' && (
-                        <Item
-                          key={child?.id}
-                          name={child.name}
-                          type={child.type as 'organization' | 'project'}
-                          category={child.category}
-                          id={child?.id}
-                          heightIndex={grandChildIndex}
-                          hasChildren={false}
-                          isFirstOfType={grandChildIndex === 0}
-                        />
-                      ),
-                  )}
-                </CollapsibleContent>
-              </Collapsible>
-            );
-          })}
+          <Item
+            name={name}
+            id={id}
+            type={type}
+            hasChildren={networks?.length > 0}
+            hasDot={networks?.length > 0}
+            className="z-40"
+          />
+          {
+            networks
+              .reverse() // We reverse the array to correct the z-index of the svgs
```

Comment:
The `reverse`  function mutates the array. Use `[...network].reverse()` instead.